src/recordpage.h

```diff
@@ -26,20 +29,12 @@ class ProcessModel;
 class ProcessFilterModel;
 class MultiConfigWidget;
 class PerfOutputWidget;
+class RecordHost;
```

Comment:
you have an include for that too here? either/or, ideally only the forward declaration